.env

```diff
@@ -90,12 +90,14 @@ LOG_LEVEL=INFO
 WORKERS_MICRONAUT_ENVIRONMENTS=control-plane
 # Cron #
 CRON_MICRONAUT_ENVIRONMENTS=control-plane
+CONNECTOR_BUILDER_SERVER_MICRONAUT_ENVIRONMENTS=control-plane
```

Comment:
note, we probably don't need this for the connector builder server

Reply:
good call. I added this because [the job persistence beans requires it](https://github.com/airbytehq/airbyte/pull/22870/files#diff-e85db0f13251e445e3894de0e5d914de6db9e37bd49e525f3220f85184d12bf5R90), but we can just remove the `@Requires`